test/powershell/Modules/Microsoft.PowerShell.Security/CertificateProvider.Tests.ps1

```diff
@@ -48,7 +48,7 @@ Describe "Certificate Provider tests" -Tags "CI" {
             param([string] $path)
             $expectedResolvedPath = Resolve-Path -LiteralPath $path
             $result = Get-Item -LiteralPath $path
-            $result | Should -Not -Be null
```

Comment:
@iSazonov The test passes when `$null` is compared to `[string]'null'` when it should fail.